src/tasks/fetcher.py

```diff
@@ -69,12 +69,17 @@ async def fetch(self):
 
                     # Filter laps by freeze time
                     if self._settings.site.freeze is not None:
-                        laps: list[Lap] = [
+                        new_laps: list[Lap] = [
                             lap
                             for lap in laps
                             if lap.timestamp <= self._settings.site.freeze
                         ]
 
+                        # If the filter removed laps, we now the scoreboard is frozen
```

Comment:
kkkkkkkkkkkkkkkkkkkkkkkk

Reply:
A bit of a hack indeed, I did not want to create a new thread just to check this :sweat_smile:

Reply:
I meant `now` -> `know` :upside_down_face: